README.md

```diff
@@ -71,9 +95,9 @@ The library is **Browser only**, it does not support the **Node Context**.
  **[Webpack Sample App repository here.](https://github.com/onfido/onfido-sdk-web-sample-app/tree/0.0.2)**
```

Comment:
This link looks to be outdated, I believe it should point the latest state of repository, that is `https://github.com/onfido/onfido-sdk-web-sample-app/`